src/components/Tables/Table.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import React, { useState, useMemo, useEffect } from "react";
+import DataTable, { TableColumn } from "react-data-table-component";
+import TableFilterBar from "./TableFilterBar";
+
+interface Props {
+    data: { [key: string]: string | number | null }[];
+    headers: { [key: string]: string | null };
+}
+
+const Table: React.FC<Props> = (props) => {
+    const [filterText, setFilterText] = useState<{ [key: string]: string }>({});
+
+    // temporary solution to fix hydration error
```

Comment:
Is this still the case? Can a more permanent solution be found?

Reply:
Or can comment be removed?

Reply:
We have tried to find multiple solutions and all working solution suggests forcing rendering the component on the client side which is what that line of code is doing. We commented it as a temporary solution as this seems to be a workaround rather than a solution.

As this method is mentioned as the first method here, https://nextjs.org/docs/messages/react-hydration-error . I think We can leave it in.